content/zh-cn/case-studies/netease/index.html

```diff
@@ -1,105 +1,174 @@
 ---
 title: 案例研究：NetEase
+linkTitle: NetEase
 case_study_styles: true
 cid: caseStudies
-css: /css/style_case_studies.css
+logo: netease_featured_logo.png
+featured: false
+
+new_case_study_styles: true
+heading_background: /images/case-studies/netease/banner1.jpg
+heading_title_logo: /images/netease_logo.png
+subheading: >
+  NetEase 如何利用 Kubernetes 支持在全球的互联网业务
+case_study_details:
+  - 公司: NetEase
+  - 位置: Hangzhou， China
```

Comment:
Use `中國杭州`, `Hangzhou, China` or  `Hangzhou，China` would be better.

```suggestion
  - 位置: Hangzhou, China
```